stripe/src/main/java/com/stripe/android/networking/AnalyticsDataFactory.kt

```diff
@@ -111,7 +111,7 @@ internal class AnalyticsDataFactory @VisibleForTesting internal constructor(
         val errorMessage = protocolErrorEvent.errorMessage
         val errorData = mapOf(
             "type" to "protocol_error_event",
-            "sdk_trans_id" to protocolErrorEvent.sdkTransactionId,
+            "sdk_trans_id" to protocolErrorEvent.sdkTransactionId?.value,
```

Comment:
it's nullable?

Reply:
Yeah, in practice it should never be null, but can't strictly guarantee that it's not-null because it's populated from JSON provided by another service.